client/src/components/Home/Home.js

```diff
@@ -3,6 +3,8 @@ import PropTypes from 'prop-types';
 import IconSelector from './IconSelector';
 import SearchBar from './SearchBar';
 import './Home.css';
+import { Link } from 'react-router-dom';
+import BigRedButton from './../../images/bigRedButton.png';
```

Comment:
I see what you mean about the margins.  Can we get rid of the other pngs that are not being used?

Reply:
yep yep